mgmtsystem_nonconformity_product/models/mgmtsystem_nonconformity_product.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+
+from odoo import fields, models
+
+
+class MgmtsystemMgmProduct(models.Model):
```

Comment:
```suggestion
class MgmtsystemNonconformity(models.Model):
```

Reply:
@mymage Please rename this file to `mgmtsystem_nonconformity.py` to match the model being inherited.